src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,24 @@
 package mate.academy
 
+import mate.academy.service.ColorSupplier
+import mate.academy.service.FigureSupplier
+
+object Main {
+    const val TOTAL_FIGURES = 6
+    const val RANDOM_FIGURE_COUNT = 3
+}
+
 fun main() {
-    // draw all figures here
+    val colorProvider = ColorSupplier()
```

Comment:
Ensure that the `ColorSupplier` class is correctly implemented and imported. It should provide random colors as expected.